engine/src/w32-ds-player.cpp

```diff
@@ -762,8 +886,13 @@ bool MCWin32DSPlayer::OpenFile(MCStringRef p_filename)
 		}
 	}
 
-	if (t_success && t_vmr9 != NULL && !MCWin32DSFilterIsConnected(t_vmr9, PINDIR_INPUT))
-		t_success = SUCCEEDED(t_graph->RemoveFilter(t_vmr9));
+	if (t_success && t_vmr9 != NULL)
+	{
+		if (MCWin32DSFilterIsConnected(t_vmr9, PINDIR_INPUT))
+			/* UNCHECKED */ MCWin32DSReconnectVideoRendererConnection(t_graph, t_vmr9);
```

Comment:
@livecodeian any reason not to assign the result to `t_success` here?

Reply:
@livecodeian just checking if you saw the question here?

Reply:
Ah I think I assumed that MCWin32DSReconnectVideoRendererConnection could fail while still leaving the graph in a valid state - i.e. still able to play the video without the mirroring working